src/field/launch_application.c

```diff
@@ -0,0 +1,1400 @@
+#include "global.h"
+#include "alph_puzzle.h"
+#include "bag_view.h"
+#include "certificates_app.h"
+#include "choose_starter_app.h"
+#include "fashion_case.h"
+#include "field_system.h"
+#include "field/launch_application.h"
+#include "field/launch_application_data.h"
+#include "friend_group.h"
+#include "frontier_data.h"
+#include "game_stats.h"
+#include "get_egg.h"
+#include "mail_misc.h"
+#include "overlay_02.h"
+#include "overlay_14.h"
+#include "overlay_16.h"
+#include "overlay_18.h"
+#include "overlay_41.h"
+#include "overlay_43.h"
+#include "overlay_54.h"
+#include "overlay_58.h"
+#include "overlay_67.h"
+#include "overlay_72.h"
+#include "overlay_73.h"
+#include "overlay_86.h"
+#include "overlay_87.h"
+#include "overlay_96.h"
+#include "overlay_99.h"
+#include "overlay_103.h"
+#include "overlay_108.h"
+#include "overlay_109.h"
+#include "overlay_111.h"
+#include "overlay_113.h"
+#include "overlay_manager.h"
+#include "overlay_trainer_card.h"
+#include "pokedex_util.h"
+#include "sys_flags.h"
+#include "trainer_data.h"
+#include "unk_020192D0.h"
+#include "unk_0202C034.h"
+#include "unk_0202C730.h"
+#include "unk_0202CA24.h"
+#include "unk_0202D230.h"
+#include "unk_0202DB34.h"
+#include "unk_02031904.h"
+#include "unk_02037C94.h"
+#include "unk_0203DFA4.h"
+#include "unk_02054E00.h"
+#include "unk_02055244.h"
+#include "unk_020552A4.h"
+#include "unk_02055418.h"
+#include "unk_0206D494.h"
+#include "unk_020755E8.h"
+#include "unk_02078E30.h"
+#include "unk_02082908.h"
+#include "unk_02087A78.h"
+#include "unk_02088288.h"
+#include "unk_0205A44C.h"
+#include "unk_020910D8.h"
+#include "unk_02092BE8.h"
+#include "unk_02097B78.h"
+#include "save_local_field_data.h"
+#include "save_pokeathlon.h"
+#include "save_special_ribbons.h"
+#include "save_trainer_card.h"
+#include "battle/battle_022378C0.h"
+#include "constants/flags.h"
+
+typedef struct Unk0203E6D4 {
```

Comment:
these shouldn't be in this file

Reply:
They're only used within this file

Reply:
understandable, I personally prefer all definitions to be in headers but they can be within the files themselves I suppose